playbooks/byo/openshift-cluster/openshift-logging.yml

```diff
@@ -1,32 +1,7 @@
 ---
-#
-# This playbook is a preview of upcoming changes for installing
-# Hosted logging on.  See inventory/byo/hosts.*.example for the
-# currently supported method.
-#
-- name: Create initial host groups for localhost
-  hosts: localhost
-  connection: local
-  become: no
-  gather_facts: no
+- include: ../../common/openshift-cluster/std_include.yml
```

Comment:
I think you are wanting to include the byo/openshift-cluster/evaluate_groups.yml created in this PR.